### PR TITLE
Added features necessary to support the new zero apply command

### PR DESCRIPTION
**Pull request by bmonkman**
Added parsing of repository var

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+GITHUB_ORG := $(shell echo ${REPOSITORY} | cut -d "/" -f 2)
+GITHUB_REPO := $(shell echo ${REPOSITORY} | cut -d "/" -f 3)
 
 run:
 	@echo "Set CIRCLECI environment variables\n"
-	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_ACCESS_KEY_ID", "value":"${AWS_ACCESS_KEY_ID}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_TOKEN}
-	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_SECRET_ACCESS_KEY", "value":"${AWS_SECRET_ACCESS_KEY}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_TOKEN}
+	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_ACCESS_KEY_ID", "value":"${AWS_ACCESS_KEY_ID}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_API_KEY}
+	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_SECRET_ACCESS_KEY", "value":"${AWS_SECRET_ACCESS_KEY}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_API_KEY}
```
Comment (cjhawkins) [inline Makefile:12]: Are AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY passed in as args now?

Comment (bmonkman) [reply Makefile:14]: Oops, no. Just missed that they were referenced in there. Added them back in.

File: zero-module.yml [hunk 2]
```diff
@@ -39,3 +39,6 @@ parameters:
   - field: stagingBackendSubdomain
     label: Staging Backend Host Name (e.g. api.)
     default: api.
+  - field: randomSeed
+    label: Random seed that will be shared between projects to come up with deterministic resource names
+    execute: uuidgen
```
Comment (davidcheung) [inline zero-module.yml:44]: uuidgen actually gives a pretty long name, alot of AWS artifacts have character limit we might not be able to use it :(

Comment (davidcheung) [reply zero-module.yml:44]: not sure what's a good alternative... `crc32 <(uuidgen)` ?

Comment (bmonkman) [reply zero-module.yml:44]: Or just `uuidgen | head -c 8`

**Pull request merged**